deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -169,6 +169,24 @@ def callback_invalid_location_pattern(location, prefix=monitoring.LOG_COLLECTOR_
     return monitoring.make_callback(pattern=msg, prefix=prefix, escape=True)
 
 
+def callback_ignoring_file(location_file, prefix=monitoring.LOG_COLLECTOR_DETECTOR_PREFIX):
+    msg = fr"DEBUG: Ignoring file '{location_file}' due to modification time"
+    return monitoring.make_callback(pattern=msg, prefix=prefix, escape=True)
+
+
+def callback_file_matches_pattern(location_pattern, location_file, prefix=monitoring.LOG_COLLECTOR_DETECTOR_PREFIX):
+    msg = fr"New file that matches the '{location_pattern}' pattern: '{location_file}'."
+    return monitoring.make_callback(pattern=msg, prefix=prefix, escape=True)
+
+
+def callback_reading_syslog_message(message, prefix=monitoring.LOG_COLLECTOR_DETECTOR_PREFIX):
+    msg = fr"DEBUG: Reading syslog message: '{message}'"
+    return monitoring.make_callback(pattern=msg, prefix=prefix, escape=True)
+
+
+def callback_read_line_from_file(n_lines, filename, prefix=monitoring.LOG_COLLECTOR_DETECTOR_PREFIX):
+    msg = fr"DEBUG: Read {n_lines} lines from {filename}"
+    return monitoring.make_callback(pattern=msg, prefix=prefix, escape=True)
```

Comment:
Docstrings are missing. Also, keep in mind following the PEP-8.

Reply:
Done 0f925459f2174454140dcf7aedc8ae418930e12a